airflow/docker_test.go

```diff
@@ -112,7 +112,7 @@ services:
   webserver:
     image: test-project-name/airflow:latest
     command: >
-      bash -c "airflow create_user -r Admin -u admin -e admin@example.com -f admin -l user -p admin && airflow webserver"
+      bash -c 'airflow create_user "$@" || airflow users create "$@"' -- -r Admin -u admin -e admin@example.com -f admin -l user -p admin && airflow webserver"
```

Comment:
Need some help here, do we need to escape `$` over here?

Reply:
```
Env file ".env" found. Loading...
ERRO[0000] Could not parse config for project t20201013104455846271 : Invalid interpolation format for key "command": "bash -c 'airflow create_user "$@" || airflow users create "$@"' -- -r Admin -u admin -e admin@example.com -f admin -l user -p admin && airflow webserver"
"
Error: Error creating docker-compose project: Invalid interpolation format for key "command": "bash -c 'airflow create_user "$@" || airflow users create "$@"' -- -r Admin -u admin -e admin@example.com -f admin -l user -p admin && airflow webserver"
"
```
yeah it doesn't work... let me think a little bit.

Reply:
@kaxil my version:
```
bash-5.0$ bash -c "args='-r Admin -u admin -e admin@example.com -f admin -l user -p admin';airflow create_user $args || airflow users create $args && airflow webserver"
[2020-10-13 08:21:13,767] {settings.py:254} INFO - settings.configure_orm(): Using pool settings. pool_size=5, max_overflow=10, pool_recycle=1800, pid=419

[2020-10-13 08:21:18,400] {__init__.py:51} INFO - Using executor LocalExecutor
[2020-10-13 08:21:18,401] {dagbag.py:403} INFO - Filling up the DagBag from /usr/local/airflow/dags
[2020-10-13 08:21:19,160] {security.py:461} INFO - Start syncing user roles.
[2020-10-13 08:21:19,454] {security.py:370} INFO - Fetching a set of all permission, view_menu from FAB meta-table
[2020-10-13 08:21:19,668] {security.py:313} INFO - Cleaning faulty perms
admin already exist in the db
[2020-10-13 08:21:21,013] {settings.py:254} INFO - settings.configure_orm(): Using pool settings. pool_size=5, max_overflow=10, pool_recycle=1800, pid=435
  ____________       _____________
 ____    |__( )_________  __/__  /________      __
____  /| |_  /__  ___/_  /_ __  /_  __ \_ | /| / /
___  ___ |  / _  /   _  __/ _  / / /_/ /_ |/ |/ /
 _/_/  |_/_/  /_/    /_/    /_/  \____/____/|__/
[2020-10-13 08:21:25,576] {__init__.py:51} INFO - Using executor LocalExecutor
[2020-10-13 08:21:25,577] {dagbag.py:403} INFO - Filling up the DagBag from /dev/null
^
```